sycl/test-e2e/Basic/interop/Interop_level_zero_backend.cpp

```diff
@@ -0,0 +1,85 @@
+// RUN: %if any-device-is-level_zero %{ %{build} -isystem %sycl_include -DBUILD_FOR_L0 -o %t-l0.out %}
```

Comment:
Is `BUILD_FOR_L0` still used?  I think it can be dropped, no?

Reply:
No, dropped. Thanks.